index.html

```diff
@@ -2,394 +2,275 @@
-
-        <section class="let">
-          <h4>LET'S CONNECT</h4>
-        </section>
-
-        <section class="apps">
-          <ul>
-            <li class="ap-1">
-              <a
-                href="https://twitter.com/abuyiag"
-                target="_blank"
-                rel="noopener noreferrer"
-                ><img src="imgs/cat.png" alt="cat chat"
-              /></a>
+        </ul>
+        <li class="frameworks">
+          <div class="user-tool-d">
+            <h3>Frameworks</h3>
+            <img class="arrow-down" src="img/arrow-head.png" alt="Arrow-head"> 
+          </div>
+          <hr />
+        </li>
+        <li class="skills">
+          <div class="user-tool-d">
+            <h3>Skills</h3>
+            <img class="arrow-down" src="img/arrow-head.png" alt="Arrow-head">
+          </div>
+          <hr />
+        </li>
+      </ul>
+    </section>
+    <section class="contact-form">
+      <div class="contact-form-holder">
+        <h2 class="contact-title">Contact Me</h2>
+
+        <p class="p-pargraph">
+          If you have an application you are interested in developing, a feature
+          that you need built or a work-section-li that needs coding. I’d love to help
+          with it
+        </p>
+
+        <form
+          action=" https://formspree.io/f/xqkwojbj"
+          class="form-contact"
+          method="post"
+        >
+          <ul class="ul-form">
+            <li class="li-form">
+                <input class= "form-name"
+                type="text"
+                placeholder="Vincent"
+                name="name"
+                id="name_form"
+                maxlength="30"
+                required
+              />
             </li>
-            <li class="ap-2">
-              <a
-                href="https://linkedin.com/in/vincent-abuya-a1940555"
-                target="_blank"
-                rel="noopener noreferrer"
-                ><img src="imgs/linkedln.png" alt="Linkedln"
-              /></a>
+            <li class="li-form">
+              <input id="form-email"
+                type="email"
+                placeholder="abuya@zipf-tech.com"
+                name="email"
+                id="email_form"
+                pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" 
+                required
+              />
             </li>
-            
-            <li class="ap-3">
-              <a
-                href="https://twitter.com/abuyiag"
-                target="_blank"
-                rel="noopener noreferrer"
-                ><img src="imgs/ychat.png" alt="Y Chat"
-              /></a>
+            <li class="li-form">
+              <textarea id= "form-textarea"
+                name="message"
+                placeholder="Write your message here"
+                id="message"
+                cols="30"
+                rows="10"
+                maxlength="500"
+                required
+              ></textarea>
             </li>
```

Comment:
- [ ] On the mobile screen, input fields are breaking your design causing horizontal scrolling, also there is a black border around the input fields, kindly investigate the issue making sure it matches the Figma design as much as possible.
![image](https://user-images.githubusercontent.com/33767951/140615219-ffea2337-7e22-42c0-b58b-d225c63ca792.png)